sui/src/wallet_commands.rs

```diff
@@ -519,6 +470,42 @@ impl WalletCommands {
             WalletCommands::ActiveAddress {} => {
                 WalletCommandResult::ActiveAddress(context.active_address().ok())
             }
+            WalletCommands::CreateExampleNFT {
+                name,
+                description,
+                url,
+                gas,
+                gas_budget,
+            } => {
+                let args_json = json!([
+                    unwrap_or(name, EXAMPLE_NFT_NAME),
+                    unwrap_or(description, EXAMPLE_NFT_DESCRIPTION),
+                    unwrap_or(url, EXAMPLE_NFT_URL)
+                ]);
+                let mut args = vec![];
+                for a in args_json.as_array().unwrap() {
+                    args.push(SuiJsonValue::new(a.clone()).unwrap());
+                }
+                let (_, effects) = call_move(
+                    &ObjectID::from_hex_literal("0x2").unwrap(),
+                    &Identifier::new("ExampleNFT").unwrap(),
+                    &Identifier::new("mint").unwrap(),
+                    &[],
+                    gas,
+                    &gas_budget.unwrap_or(1000),
```

Comment:
I think this should be fine because https://github.com/MystenLabs/sui/blob/b0d51b5301d3022cca171ff6f97874c4b5b4fe0c/sui/src/wallet_commands.rs#L596 will try to find a gas object that's greater than 1000, and locally it costs `GasCostSummary { computation_cost: 761, storage_cost: 40, storage_rebate: 0 }`, which is close enough to 1000. If the command still fails, the user can always pass in the budget manually.